server.js

```diff
@@ -284,6 +284,63 @@ camp.notfound(/.*/, function(query, match, end, request) {
 
 // Vendors.
 
+// JIRA integration
+camp.route(/^\/jira\/(http(?:s)?)\/(.+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you add a badge-specific prefix, in case we have multiple JIRA badges in the future? Maybe `jira/issue`?